charts/bitbucket-runner-autoscaler/templates/_helpers.tpl

```diff
@@ -164,3 +166,57 @@ Define runner node selector
 {}
 {{- end }}
 {{- end }}
+
+{{/*
+Return the controller image name
+*/}}
+{{- define "bitbucketRunnerAutoscaler.controllerImage" -}}
+{{- $registryName := .Values.controller.image.registry -}}
+{{- $repositoryName := .Values.controller.image.repository -}}
+{{- $tag := .Values.controller.image.tag | toString -}}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+
+{{/*
+Return the cleaner image name
+*/}}
+{{- define "bitbucketRunnerAutoscaler.cleanerImage" -}}
+{{- $registryName := .Values.cleaner.image.registry -}}
+{{- $repositoryName := .Values.cleaner.image.repository -}}
+{{- $tag := .Values.cleaner.image.tag | toString -}}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+
+{{/*
+Return the runner image name
+*/}}
+{{- define "bitbucketRunnerAutoscaler.runnerImage" -}}
+{{- $registryName := .Values.runner.image.registry -}}
+{{- $repositoryName := .Values.runner.image.repository -}}
+{{- $tag := .Values.runner.image.tag | toString -}}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+
+{{/*
+Return the dind image name
+*/}}
+{{- define "bitbucketRunnerAutoscaler.dindImage" -}}
+{{- $registryName := .Values.runner.dind.registry -}}
+{{- $repositoryName := .Values.runner.dind.repository -}}
+{{- $tag := .Values.runner.dind.tag | toString -}}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+
+{{- define "bitbucketRunnerAutoscaler.pdbName" -}}
+{{- $component := .component -}}
+{{- $context := .context -}}
+{{- printf "%s-%s-pdb" (include "bitbucketRunnerAutoscaler.fullname" $context) $component -}}
+{{- end }}
+
+{{- define "bitbucketRunnerAutoscaler.templateSecret" -}}
```

Comment:
Is there any benefit of having this helper? It just wraps a value dump.

Reply:
you are right no need at all. Removed